test/row.ts

```diff
@@ -766,21 +767,60 @@ describe('Bigtable/Row', () => {
   describe('exists', () => {
     it('should not require gaxOptions', done => {
       sandbox.stub(row, 'getMetadata').callsFake(gaxOptions => {
-        assert.deepStrictEqual(gaxOptions, {});
+        assert.deepStrictEqual(gaxOptions, {
+          filter: [
+            {
+              row: {
+                cellLimit: 1,
+              },
+            },
+            {
+              value: {
+                strip: true,
+              },
+            },
+          ],
+        });
         done();
       });
       row.exists(assert.ifError);
     });
 
-    it('should pass gaxOptions to getMetadata', done => {
+    it('should add filter to the read row options', done => {
       const gaxOptions = {};
       sandbox.stub(row, 'getMetadata').callsFake(gaxOptions_ => {
-        assert.strictEqual(gaxOptions_, gaxOptions);
+        assert.deepStrictEqual(gaxOptions_, {
+          filter: [
+            {
+              row: {
+                cellLimit: 1,
+              },
+            },
+            {
+              value: {
+                strip: true,
+              },
+            },
+          ],
+        });
         done();
       });
       row.exists(gaxOptions, assert.ifError);
     });
 
+    it('should pass gaxOptions to getMetadata', done => {
+      const gaxOptions = {
+        testProperty: true,
+      } as CallOptions;
+
+      sandbox.stub(row, 'getMetadata').callsFake(gaxOptions_ => {
+        assert('testProperty' in gaxOptions_);
```

Comment:
Can you switch to `assert.strictEqual(gaxOptions_.testProperty, gaxOptions.testProperty);`?

Reply:
Done!!

Reply:
Thank you!